example_cases/SPARC_PRD/input.yaml

```diff
@@ -166,3 +168,10 @@ fraction_of_P_SOL_to_divertor: 0.6
 kappa_e0: 2600.0
 # Calculate P_rad_SOL such that the target electron temperature in eV reaches this value (for FixedTargetElectronTemp)
 target_electron_temp: 25.0
+
+# Which species should be injected to cool the scrape-off-layer?
+edge_impurity_species: Argon
+# Factor applied to the result of the Lengyel model to give an absolute low-Z impurity concentration.
+lengyel_overestimation_factor: 4.3
+# Ratio of edge impurity concentration in the edge to in the core (c_edge / c_core)
+edge_impurity_enrichment: 5.0
```

Comment:
Should `SPARC_PRD` evolve and have these added features or be fixed to use the same methods it originally was defined with?

Reply:
Ok to move this to a new issue 👍 Would you mind creating that and linking it here?

Reply:
#27